src/Microsoft.IdentityModel.Clients.ActiveDirectory/Features/WinCommon/Logger.cs

```diff
@@ -38,7 +38,22 @@ static Logger()
 
         internal override void DefaultLog(LogLevel logLevel, string message)
         {
-            AdalEventSource.Error(message);
+            if (logLevel == LogLevel.Information)
+            {
+                AdalEventSource.Information(message);
+            }
+            else if (logLevel == LogLevel.Verbose)
```

Comment:
I might be a bit picky but I would have ordered them by severity? (Verbose first, and then information, warning, and error) ?

Reply:
yes...great point - thank you